### PR TITLE
Add basic docs for writing pioneer schemas.

### PR DESCRIPTION
**Pull request by mreid-moz**

### CODE REVIEW

**Comment by mreid-moz** [comment]
I'm not 100% sure this is correct, particularly the description about the outer document format vs. the payload and what incoming pioneer pings actually look like.  From the example I saw during #209, the incoming ping in its entirety looks something like this:

```json
{
 "encryptedData": "eyJh...sAQ",
 "encryptionKeyId": "pioneer-20170905",
 "pioneerId": "1076d9e9-152a-465d-85bf-d3ac056beb8d",
 "studyName": "jestr-pioneer-shield-study@pioneer.mozilla.org",
 "schemaName": "shield-study-addon",
 "schemaVersion": 3
}
```

But I'm not sure if that matches what is actually sent out by the client.

**Comment by motin** [comment]
Looks good! In case there are any shield study authors coming across this documentation, don't forget to mention that they don't have to add their own schemas, instead they can send their custom payloads using [browser.study.sendTelemetry(payload)](https://github.com/mozilla/shield-studies-addon-utils/blob/master/webExtensionApis/study/api.md#browserstudysendtelemetry-payload-), given that the payload adheres to the `data.attributes` property in the [`shield-study-addon`](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/shieldStudyAddonPayload.3.schema.json#L49) schema. This applies to both "ordinary" shield telemetry and Pioneer studies using shield-studies-addon-utils.

**Comment by mreid-moz** [comment]
@motin I've added a note to the README.

File: README.pioneer.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# About Pioneer Schemas
+
+**Note:** If you are writing a Pioneer study using [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils/pull/263), you do not need to do anything special and data sent using `browser.study.sendTelemetry(payload)` should "just work", as long as the payload adheres to the [`shield-study-addon`](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/shieldStudyAddonPayload.3.schema.json) schema.
```
Comment (motin) [inline README.pioneer.md]:
```suggestion
**Note:** If you are writing a Pioneer study using [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils/pull/263), you do not need to do anything special and data sent using `browser.study.sendTelemetry(payload)` should "just work", as long as the payload adheres to the `data.attributes` property in the [`shield-study-addon`](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/shieldStudyAddonPayload.3.schema.json) schema.
```

Comment (motin) [reply README.pioneer.md]: If this is too long, we can move the longer version of this note to [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils) and merely link to those docs here.

Comment (mreid-moz) [reply README.pioneer.md]: More detail is good. Thanks!

**Pull request merged**